spec/models/user_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe Account::User, type: :model do
+  context 'validation tests' do
+    let!(:user) { build(:user) }
+
+    describe 'factory' do
+      it { is_expected.to be_truthy }
+    end
+
+    describe '#first_name' do
+      it { is_expected.to validate_presence_of(:first_name) }
+
+      it 'does not save the object with 1 letter' do
+        user.update(first_name: "A")
```

Comment:
you don't need to update user in the database, to test if it valid. Use just  `user.first_name = 'A'` and `user.validate` check errors messages